mathics/Makefile

```diff
@@ -1,13 +0,0 @@
-put:
```

Comment:
Not really objecting to the removal of this makefile, but is this change logically related to Manipulate? Would it be better as a separate PR?